server/models/index.js

```diff
@@ -28,7 +27,10 @@ Object.keys(db).forEach(modelName => {
   }
 });
 
+if (env === "production" || env === "PRODUCTION") {
```

Comment:
What does this do and is one for production and the other SUPER PRODUCTION?  Also why for only production what about test and development?